components/docs-chef-io/content/automate/chef_automate_profile.md

```diff
@@ -0,0 +1,47 @@
++++
+title = "Profile"
+
+draft = false
+
+gh_repo = "automate"
+[menu]
+  [menu.automate]
+    title = "Profile"
+    parent = "automate/settings"
+    identifier = "automate/settings/chef_automate_profile.md Profile"
+    weight = 120
++++
+
+## Overview
+
+The Chef Automate **Profile** help you set your personal information about the entities. You can set the _Display Name_, _Time Format_ and reset your _Password_.
+
+{{< figure src="/images/automate/chef-automate-profile.png" alt="Profile Section in Chef Automate">}}
+
+Select **Profile** to view the details with respect to your _username_. The **Details** tab lets you update your _Display Name_ and the _Time Format_, whereas the **Reset Password** tab lets you update your password.
+
+### Update the Display Name
+
+You can update the display name for your Chef Automate in the bar.
+
+{{< figure src="/images/automate/display-name-in-profile.png" alt="Display Name of Chef Automate">}}
+
+The Display Name bar should not be kept empty or else it throws error.
+
+{{< figure src="/images/automate/error-in-display-name-bar.png" alt="Error in Display Name">}}
+
+### Update the Date/Time Format
+
+You can select the date and time format amongst the five formats provided in Chef Automate.
```

Comment:
```suggestion
You can select one of five date and time formats.
```

Are we changing the time zone or just the formatting? It isn't clear to me what the user can do.

Reply:
Also, it doesn't look like you removed the old section from the docs.

Reply:
We can just change the formatting